src/runtime/components/CldVideoPlayer.vue

```diff
@@ -195,6 +198,8 @@ const handleOnLoad = () => {
       ...logoOptions,
       hideContextMenu,
       pictureInPictureToggle,
+      ...useRuntimeConfig().public.cloudinary.config,
```

Comment:
i think this need to spread the `config.url` in this instance, but per my Discord message, i think exposing `cloudinary.url` and spreading that directly might be a better experience